bookwyrm/views/rss_feed.py

```diff
@@ -177,3 +178,61 @@ def item_link(self, item):
     def item_pubdate(self, item):
         """publication date of the item"""
         return item.published_date
+
+
+class RssShelfFeed(Feed):
+    """serialize a shelf activity in rss"""
+
+    description_template = "rss/edition.html"
+
+    def item_title(self, item):
+        """render the item title"""
+        if hasattr(item, "pure_name") and item.pure_name:
+            return item.pure_name
+        authors = item.authors
+        authors.display_name = f"{item.author_text}:"
+        template = get_template("rss/title.html")
+        return template.render({"user": authors, "item_title": item.title}).strip()
+
+    def get_object(
+        self, request, shelf_identifier, username
+    ):  # pylint: disable=arguments-differ
+        """the shelf that gets serialized"""
+        user = get_user_from_username(request.user, username)
+        # always get privacy, don't support rss over anything private
+        # get the SHELF of the object
+        shelf = get_object_or_404(
+            user.shelf_set,
+            identifier=shelf_identifier,
+            privacy__in=["public", "unlisted"],
+        )
+        shelf.raise_visible_to_user(request.user)
+        return shelf
+
+    def link(self, obj):
+        """link to the shelf"""
+        return obj.local_path
+
+    def title(self, obj):
+        """title of the rss feed entry"""
+        return _(f"Books added to {obj.name}")
```

Comment:
Since this is the title RSS readers will show, how about including the shelf owner in a concise manner:

```suggestion
        return _(f"{obj.user.name}’s {obj.name} shelf")
```